db/migrate/20240306082827_create_transaction_categories.rb

```diff
@@ -0,0 +1,15 @@
+class CreateTransactionCategories < ActiveRecord::Migration[7.2]
+  def change
+    create_table :transaction_categories, id: :uuid do |t|
+      t.string "name", null: false
+      t.string "icon", null: false
```

Comment:
I just checked with our designer and found out we _won't_ be using icons for categories.  So we can remove this.

Here's what it will look like:

![CleanShot 2024-03-07 at 10 18 37](https://github.com/maybe-finance/maybe/assets/16676157/78d7b1d3-d0d9-4dbb-8ef7-a5d408c0d799)